classes/01class/exercises/c01-git03/igorjosesantos/c01-git03-1.txt

```diff
@@ -0,0 +1,11 @@
+Q: Did you notice any difference in the size of the repo before and after adding the big file?
+A: Yeah. It became approximately 20MB bigger
+
+Q: What is the reason for this problem?
+A: Git stores a copy of the additions/changes inside the .git folder
+
+Q: How do you remove something from Git history? Which commands would you use? What are the consequences?
+A: Using the commands below:
+
+git reflog expire --expire=now --all
```

Comment:
`reflog` is available only in your machine. 

Suppose your laptop caught on fire and you got a new one. You just cloned the repo and need to solve the problem.

Which commands you can use?

Reply:
Done!